.github/workflows/srt_nuopc.yml

```diff
@@ -0,0 +1,195 @@
+# events but only for the master branch
+on:
+  push:
+    branches: [ master, maint-5.6 ]
+  pull_request:
+    branches: [ master, maint-5.6 ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9]
+    env:
+      CC: mpicc
+      FC: mpifort
+      CXX: mpicxx
+      CPPFLAGS: "-I/usr/include -I/usr/local/include"
+      # Versions of all dependencies can be updated here
+      PNETCDF_VERSION: pnetcdf-1.12.2
+      NETCDF_FORTRAN_VERSION: v4.5.2
+      MCT_VERSION: MCT_2.11.0
+      PARALLELIO_VERSION: pio2_5_4
+      NETCDF_C_PATH: /usr
+      NETCDF_FORTRAN_PATH: ${HOME}/netcdf-fortran
+      PNETCDF_PATH: ${HOME}/pnetcdf
+      ESMF_VERSION: ESMF_8_1_1
+      CIME_MODEL: cesm
+      CIME_DRIVER: nuopc
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+
+      - uses: actions/checkout@v2
+
+      - name: share checkout
+        uses: actions/checkout@v2
+        with:
+          repository: ESCOMP/CESM_share
+          path:  share
+
+      - name: cpl7 checkout
+        uses: actions/checkout@v2
+        with:
+          repository: ESCOMP/CESM_CPL7andDataComps
+          path:  components/cpl7
+
+      - name: mct checkout
+        uses: actions/checkout@v2
+        with:
+          repository: MCSclimate/MCT
+          path:  libraries/mct
+          ref: ${{ env.MCT_VERSION }}
+
+      - name: cmeps checkout
+        uses: actions/checkout@v2
+        with:
+          repository: ESCOMP/CMEPS
+          path:  components/cmeps
+
+      - name: cdeps checkout
+        uses: actions/checkout@v2
+        with:
+          repository: ESCOMP/CDEPS
+          path:  components/cdeps
+
+      - name: fox checkout
+        uses: actions/checkout@v2
+        with:
+          repository: ESMCI/fox
+          path:  components/cdeps/fox
+
+      - name: parallelio checkout
+        uses: actions/checkout@v2
+        with:
+          repository: NCAR/ParallelIO
+          path:  libraries/parallelio
+          ref: ${{ env.PARALLELIO_VERSION }}
+
+      - id: load-env
+        run: |
+          sudo apt-get update
+          sudo apt-get install libxml2-utils pylint wget gfortran openmpi-bin netcdf-bin libopenmpi-dev cmake libnetcdf-dev
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: cache pnetcdf
+        id: cache-pnetcdf
+        uses: actions/cache@v2
+        with:
+          path: ~/pnetcdf
+          key: ${{ runner.os }}-nuopc-${{ env.PNETCDF_VERSION}}-pnetcdf-redo
```

Comment:
ultimately the key here should be the same as the key for the mct build

Reply:
pnetcdf keys are now the same.